### DISCUSSION TITLE
How to disable checking unused variable in catch(e) in ES5?

### DISCUSSION THREAD

**Discussion by jcubic** [eslint/eslint#18685] [Q&A] [answered]
This is silly, I have version ES5 that require to use variable in catch, yet ESLint throw error about unused var.

How to turn this off?

**Comment by fasttime** [comment, marked as answer]
I suppose the error is being reported by the [`no-unused-vars`](https://eslint.org/docs/latest/rules/no-unused-vars) rule. If so, you can set option `caughtErrors` to `"none"` to disable the error for catch parameters, e.g.

```js
// eslint.config.js
module.exports = [
    {
        languageOptions: {
            ecmaVersion: 5,
            sourceType: "commonjs"
        }
    },
    {
        rules: {
            "no-unused-vars": [
                "error",
                {
                    caughtErrors: "none"
                }
            ]
        }
    }
];
```

[**EXAMPLE**](https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLXVudXNlZC12YXJzOiBbXG4gIFwiZXJyb3JcIiwge1xuICAgIFwiY2F1Z2h0RXJyb3JzXCI6IFwibm9uZVwiXG4gIH1dXG4qL1xuXG50cnkge1xuICBkb1NvbWV0aGluZygpO1xufSBjYXRjaCAoZXJyb3IpIHtcbiAgLy8gaWdub3JlIGVycm9yXG59Iiwib3B0aW9ucyI6eyJydWxlcyI6e30sImxhbmd1YWdlT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6NSwic291cmNlVHlwZSI6ImNvbW1vbmpzIiwicGFyc2VyT3B0aW9ucyI6eyJlY21hRmVhdHVyZXMiOnt9fX19fQ==)

**Reply by nickolay** [reply]
Note that "no-unused-vars" is [not what you want for typescript projects](https://stackoverflow.com/questions/57802057/eslint-configuring-no-unused-vars-for-typescript), which confusingly have their own rule to configure: `export default ts.config(
.... {rules: : { '@typescript-eslint/no-unused-vars': ['warn', { caughtErrors: 'none' }] ....`
